### PR TITLE
clean up, release and use

### PR DESCRIPTION
**Pull request by turadg**
Adds `getInstanceBoardId` to support some a3p-integration use cases. cc @Chris-Hibbert 

Also bumps deps for hygiene, cuts a [1.1.1 release](https://www.npmjs.com/package/@agoric/synthetic-chain/v/0.1.1) and DRYs some proposal code to use that release.

an export was missing from 0.1.1 so I also cut 1.2.0, which is a more correct version number since it adds a feature.

Validated in https://github.com/Agoric/agoric-sdk/pull/9977

### CODE REVIEW

File: packages/synthetic-chain/package.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
   "scripts": {
     "prepare": "yarn build",
     "build": "tsup",
-    "test": "NODE_OPTIONS='--loader=tsx --no-warnings' ava"
+    "test": "ava"
```
Comment (dckc) [inline packages/synthetic-chain/package.json:19]: :clap: less cruft

File: proposals/29:upgrade-9/sanity.test.js [hunk 2]
```diff
@@ -3,11 +3,7 @@ import test from 'ava';
 
 import fsp from 'node:fs/promises';
 
-import {
-  agd,
-  agops,
-  agoric,
-} from '@agoric/synthetic-chain';
+import { agd, agops, agoric } from '@agoric/synthetic-chain';
```
Comment (dckc) [inline proposals/29:upgrade-9/sanity.test.js:6]: :clap: more concise style

File: packages/synthetic-chain/test/snapshots/test-exports.ts.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
+# Snapshot report for `test/test-exports.ts`
```
Comment (dckc) [inline packages/synthetic-chain/test/snapshots/test-exports.ts.md:1]: :clap: ooh... nifty idea!

File: proposals/64:crabble-start/core-eval-support.js [hunk 4]
```diff
@@ -1,151 +0,0 @@
-// @ts-check
```
Comment (dckc) [inline proposals/64:crabble-start/core-eval-support.js:1]: :clap: whee! bye-bye cruft.

**Review by dckc — APPROVED** [review]
very nice

File: packages/synthetic-chain/src/lib/vstorage.js [hunk 5]
```diff
@@ -48,3 +48,24 @@ export const getQuoteBody = async path => {
   return JSON.parse(body.body.substring(1));
 };
 
+/**
+ *
+ * @param {string} instanceName
+ * @returns {string | null} boardId of the named instance in agoricNames
```
Comment (turadg) [inline packages/synthetic-chain/src/lib/vstorage.js:54]: This is a bug. I don't know why it passed CI. Will revisit

**Pull request merged**